frontend/src/EditTask.svelte

```diff
@@ -324,6 +332,26 @@ async function deleteTask(proceed) {
                         onFromDuplicateClick={setAssignedDateToVisible}
                         onToDuplicateClick={setDeadlineToAssigned}
                         onToRelativeClick={setRelativeDeadlineToAssigned} />
+                      {#if clazz.deadline}
+                        <div class="col-md">
+                          <div class="input-group input-group-sm col-md">
+                            <input
+                              class="form-check-input checkbox-md"
+                              type="checkbox"
+                              id="hardDeadline_{idx}"
+                              bind:checked={clazz.hard_deadline} />
+                            <label class="form-check-label check_box_label" for="hardDeadline_{idx}"
+                              >Hard Deadline</label>
+                            <button
+                              class="btn btn-sm btn-secondary"
+                              on:click|preventDefault={() =>
+                                assignHardDeadlineToAll(clazz.hard_deadline)}
+                              title="Set all hard deadlines">
```

Comment:
```suggestion
                              title="Copy hard deadline setting to all classes">
```
Otherwise it sounds like it sets (to true) the option for all classes.

Reply:
Sounds better, thanks.